demos/security_barrier_camera_demo/main.cpp

```diff
@@ -747,43 +748,84 @@ int main(int argc, char* argv[]) {
                     ie.SetConfig({{ CLDNN_CONFIG_KEY(PLUGIN_THROTTLE), "1" }}, "GPU");
                 }
             }
-
-            if ("FPGA" == device) {
-                ie.SetConfig({ { InferenceEngine::PluginConfigParams::KEY_DEVICE_ID, FLAGS_fpga_device_ids } }, "FPGA");
-            }
         }
 
         /** Per layer metrics **/
         if (FLAGS_pc) {
             ie.SetConfig({{PluginConfigParams::KEY_PERF_COUNT, PluginConfigParams::YES}});
         }
 
-        /** Graph tagging via config options**/
-        auto makeTagConfig = [&](const std::string &deviceName, const std::string &suffix) {
+        std::map<std::string, std::string> executionDevices = {{FLAGS_d, FLAGS_d}};
+        if (!FLAGS_d_va.empty()) {
+            executionDevices[FLAGS_d_va] = FLAGS_d_va;
+        }
+        if (!FLAGS_d_lpr.empty()) {
+            executionDevices[FLAGS_d_lpr] = FLAGS_d_lpr;
+        }
+
+
+        /** Replace FPGA device with MULTI if several device IDs are presented**/
+        std::vector<std::string> deviceIdStr;
+        if (!FLAGS_fpga_device_ids.empty()) {
+            std::stringstream strm{FLAGS_fpga_device_ids};
+            while (strm.good()) {
+                std::string substr;
+                std::getline(strm, substr, ',');
+                deviceIdStr.push_back(substr);
+            }
```

Comment:
You can reuse https://github.com/opencv/open_model_zoo/blob/a92e9f30fcd76f91d238124f631a5996f18eb4b5/demos/common/samples/args_helper.hpp#L94